source/includes/_quickstart-settlement.md

```diff
@@ -168,20 +169,21 @@ The following example includes a single investment from a single Investor:
 
 ```json
 {
-    "id": "investment1",
-    "investments": [
-        {
-            "accountType": "ISA",
-            "amount": {
-                "amount": 10000.00,
-                "currency": "GBP"
-            },
-            "clientId": "<client-id>",
-            "id": "client-investment-1"
-        }
-    ],
-    "paymentDestinationId": "<payment-destination-id>",
```

Comment:
this field is deprecated and the `bankAccountId` was not even present in the docs (in the quickquide)

also the validation on the dto does not work properly (javax api annottions are ignored) but i'm not gonna investigate atm